Np: 500
Nc: 100
T: 0.02
delta: 0.1

Q_x: 30.0
Q_y: 30.0
Q_yaw: 30.0

R_vx: 1
R_vy: 1
R_w: 1

Final_Position_Weights: 100.0

vx_min: -30
vy_min: -30
omega_min: -30

vx_max: 30
vy_max: 30
omega_max: 30

delta_vx_min: -1
delta_vy_min: -1
delta_omega_min: -1

delta_vx_max: 1
delta_vy_max: 1
delta_omega_max: 1

Using_Ulack_Variables: true
rho: 1.0

speed_ratio: 5

stop: false